### PR TITLE
Update puppeteer-core to 9.1.1

### PR DESCRIPTION
**Pull request by atjn**
This is a new version of #635, now based on a local branch

This updates `puppeteer-core` to `9.1.1` (the last version that still supports node v10).

`puppeteer-core` is now natively typed, which has resulted in a few renamed interfaces and a single bug that needs a workaround :)

### CODE REVIEW

**Review by onderceylan — DISMISSED** [review]
LGTM. Thanks a lot for your help @atjn!

**Comment by onderceylan** [comment]
I'd suggest making sematic-release configuration changes first so that this PR can trigger a release after the merge. Do you mind @atjn if we keep it without merging a little more?

**Comment by atjn** [comment]
Awesome! No, we can definitely wait until semantic release works properly. I also need to make a quick amendment because I just pushed a Prettier update :)

**Comment by atjn** [comment]
@onderceylan you'll have to review it again, but the only thing I fixed was some new Prettier errors.

**Review by onderceylan — APPROVED** [review]
LGTM, thank you for prettier correction!

**Review by onderceylan — APPROVED** [review]

**Pull request merged**
